CHANGELOG.md

```diff
@@ -1,8 +1,15 @@
 ## master / unreleased
- - [BUGFIX] Calling `Close` more than once on a querier returns an error instead of a panic.
- - [ENHANCEMENT] Add (again) FromData function to easily create a chunk from bytes.
 
 
+## 0.8.0
+ - [BUGFIX] Calling `Close` more than once on a querier returns an error instead of a panic.
+ - [BUGFIX] No more panic and recover nicely when running out of disk space.
```

Comment:
`Don't panic and recover` seems better grammar wise.

Reply:
Thanks updated.